lib/agents/map-control-orchestrator.tsx

```
@@ -0,0 +1,472 @@
+                     q: text
+                 },
+                 connectedAccountId: this.options.connectionId
+             });
+            break;
+
+          case 'generate_map_link':
+            // Generate a static map link or similar
+            // We can return a result object with the link
+            const link = `https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/${encodeURIComponent(text)}/600x400?access_token=${process.env.MAPBOX_ACCESS_TOKEN}`;
+            result = {
+                url: link,
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)